src/pokemon.c

```diff
@@ -5274,10 +5273,7 @@ void AdjustFriendship(struct Pokemon *mon, u8 event)
         friendship += mod;
         if (mod > 0)
         {
-            if (GetMonData(mon, MON_DATA_POKEBALL, NULL) == ITEM_LUXURY_BALL)
-                friendship++;
-            if (GetMonData(mon, MON_DATA_MET_LOCATION, NULL) == GetCurrentRegionMapSectionId())
-                friendship++;
+            ApplyFriendshipBonuses(mon,&friendship);
         }
```

Comment:
```suggestion
            ApplyFriendshipBonuses(mon, &friendship);
```

Reply:
Didn't see this before I pushed but I think this is covered now?